examples/rollup/src/index.js

```diff
@@ -22,17 +22,21 @@ const clientSecret = await fetchClientSecret();
 const stripeConnect = await loadConnect();
 if (clientSecret) {
   const connectInstance = stripeConnect.initialize({
-    publishableKey: "{{publishable key}}",
+    publishableKey: "",
     clientSecret: clientSecret,
     appearance: {
-      colorPrimary: "#228403",
+      variables: {
+        colorPrimary: "#FF3333",
+      },
     }
   });
-  const payments = connectInstance.create("payments");
-  document.getElementById("payments").append(payments);
+  const onboarding = connectInstance.create("account-onboarding");
```

Comment:
fyi I ran into this error when testing locally
```You are trying to create an Account Session for an account that is not supported.```
which I suppose is due to my test account having stripe own onboarding, and this is working as intended

Will this be common for others to run into this issue?

Reply:
This example will be merged into master/GA, where the only component that is in GA will be account onboarding. Currently we only support custom accounts for onboarding, so if users try to use a different account type they will run into that error. It's a good callout though! I will add a comment linking to our docs that they need to test with a US custom account